fbfmaproom/fbfmaproom.py

```diff
@@ -745,11 +745,37 @@ def calculate_bounds(pt, res, origin):
 def country(pathname: str) -> str:
     return pathname.split("/")[2]
 
+APP.clientside_callback(
+    """
+    function update_layout(disp, zoom) {
+        var lclass = ""
+        var rclass = ""
+     
+        if (!disp.includes("Map")) {
+            lclass = "d-none"
+        }
+
+        if (!disp.includes("Table")) {
+            rclass = "d-none"
+        }
+
+        return [ lclass, rclass, zoom ]
+    }
+
+    """,
+    Output("lcol", "className"),
+    Output("rcol", "className"),
+    Output("map", "zoom"),
```

Comment:
I think you must not have this commit in the version that's running on devi. When I run this version, I get an error in the js console about multiple callbacks trying to update the same output, which is not allowed.